src/DssSpellCollateral.sol

```diff
@@ -30,99 +30,87 @@ contract DssSpellCollateralAction {
     // A table of rates can be found at
     // https://ipfs.io/ipfs/QmVp4mhhbwWGTfbh2BzwQB9eiBrQBKiqcPRZCaAxNUaar6
     //
-
+    uint256 internal constant ONE_FIVE_PCT_RATE = 1000000000472114805215157978;
+    
     // --- Math ---
-    // uint256 constant THOUSAND   = 10 ** 3;
+    uint256 constant THOUSAND   = 10 ** 3;
     uint256 constant MILLION    = 10 ** 6;
     // uint256 constant BILLION    = 10 ** 9;
 
-    bytes32 constant RWA007_A                 = "RWA007-A";
-    uint256 constant RWA007_A_AUTOLINE_AMOUNT = 250 * MILLION;
-    uint256 constant RWA007_A_AUTOLINE_GAP    =  50 * MILLION;
-    uint256 constant RWA007_A_AUTOLINE_TTL    = 1 weeks;
-
     // --- DEPLOYED COLLATERAL ADDRESSES ---
-    // address constant XXX                  = 0x0000000000000000000000000000000000000000;
-    // address constant PIP_XXX              = 0x0000000000000000000000000000000000000000;
-    // address constant MCD_JOIN_XXX_A       = 0x0000000000000000000000000000000000000000;
-    // address constant MCD_CLIP_XXX_A       = 0x0000000000000000000000000000000000000000;
-    // address constant MCD_CLIP_CALC_XXX_A  = 0x0000000000000000000000000000000000000000;
-
-    // --- Offboarding: Current Liquidation Ratio ---
-    // uint256 constant CURRENT_XXX_A_MAT              =  XYZ * RAY / 100;
-
-    // --- Offboarding: Target Liquidation Ratio ---
-    // uint256 constant TARGET_XXX_A_MAT               =  XYZ * RAY / 100;
-
+    address internal constant RETH                 = 0xae78736Cd615f374D3085123A210448E74Fc6393;
+    address internal constant PIP_RETH             = 0xeE7F0b350aA119b3d05DC733a4621a81972f7D47;
+    address internal constant MCD_JOIN_RETH_A      = 0xC6424e862f1462281B0a5FAc078e4b63006bDEBF;
+    address internal constant MCD_CLIP_RETH_A      = 0x27CA5E525ea473eD52Ea9423CD08cCc081d96a98;
+    address internal constant MCD_CLIP_CALC_RETH_A = 0xc59B62AFC96cf9737F717B5e5815070C0f154396;
```

Comment:
these contracts were all deployed with: `0x45ea4fadf8db54df5a96774167547893e0b4d6a5` we should probably add that to `addresses_deployers.sol` too